Content.Shared/_Starlight/Economy/PriceCategoryPrototype.cs

```diff
@@ -0,0 +1,19 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Economy;
+
+/// <summary>
+/// Prototype that defines a price category range for price generation
+/// </summary>
+[Prototype("priceCategory")]
+public sealed partial class PriceCategoryPrototype : IPrototype
+{
+    [IdDataField]
+    public string ID { get; private set; } = default!;
+
+    [DataField("min", required: true)]
+    public int Min;
```

Comment:
```suggestion
    [DataField("min")]
    public int Min = 0;
```

You can make this not required tbh